### ISSUE TITLE
"Extra" data of LocalNotification is not used when creating native notifications

### ISSUE THREAD

**Issue by JackD111** [ionic-team/capacitor#1441]
Hello,

I've started playing around and testing local notifications and noticed that the `extra` property of `LocalNotification` is not used when scheduling notifications. I've checked this in https://github.com/pegler/capacitor/blob/8039f1bd7210a3ce9be981218c9580c0ace93017/ios/Capacitor/Capacitor/Plugins/LocalNotifications.swift#L156 where the notification is created.

Is there a reason, that the `userInfo` property of the returned `UNNotificationContent` is not set with the extra data? This could than later be retrieved in the `localNotificationActionPerformed` listener where currently only the notification id is passed for the notification request.

Right now, the only way to find out what extra data the notifications were scheduled with would be to store created notifications in storage and retrieve them when an action has been performed to lookup the extra data.

**Comment by jcesarmobile** [comment]
Any reason to link a fork instead of our repo?

Check our master, you'll see that extra is properly handled now. https://github.com/ionic-team/capacitor/blob/master/ios/Capacitor/Capacitor/Plugins/LocalNotifications.swift#L174

**Comment by JackD111** [comment]
Sorry, my bad. I was looking trough issues and from there must have gone to a fork instead of the main repo. The reason I created the issue in the first place is because my `notificationRequest` in the `localNotificationActionPerformed` handler still only contains the notification id.

Checking the commit history shows that that feature was implemented 19 days ago but since then, there was no new capacitor release. How can I get the latest capacitor version available on git into my project?
